### ISSUE TITLE
custom icon for custom package

### ISSUE THREAD

**Issue by ksobon** [DynamoDS/Dynamo#8748]
Is there a way to assign a custom icon to a package in new Dynamo 2.0 ?

![image](https://user-images.githubusercontent.com/529781/38621726-74b9665c-3d6f-11e8-85ea-270d734739ab.png)

**Comment by mjkkirschner** [comment]
yes, it should work the same as before - requiring a customization.dll Did you do anything else special to get your icons showing up in 1.3?

**Comment by ksobon** [comment]
Yeah so the customization DLL worked a certain way when you named the resources after class names or if it were a custom components then you would use an ID of that component etc. There was never a method for setting the "package icon". Am I missing something?

**Comment by andydandy74** [comment]
@ksobon I think @mjkkirschner is referring to the default icon for custom nodes:
https://github.com/DynamoDS/Dynamo/wiki/Creating-Icons-for-Custom-Nodes#nb-default-icon
Works for me in Clockwork and you seem to have defined one for Mandrill, too:
![grafik](https://user-images.githubusercontent.com/3014437/38764934-ecea09e2-3fb8-11e8-9511-a4ef3e263d68.png)

**Comment by ksobon** [comment]
@andydandy74 i thought that's how you define the default icon for ALL nodes and this works so that all nodes have that one single icon. Also the Mandrill icon above is from one of the nodes in there, question is how did Dynamo decide to use that?

**Comment by andydandy74** [comment]
@ksobon Interesting. Looks like it takes the icon of the first node in the package's library structure.
![grafik](https://user-images.githubusercontent.com/3014437/38769437-7219df4e-4003-11e8-8078-351d51994025.png)
Maybe still worth a try to experiment with the default icon, though.

**Comment by sixtysecondrevit** [comment]
> 
> 
> @ksobon Interesting. Looks like it takes the icon of the first node in the package's library structure.
> ![grafik](https://user-images.githubusercontent.com/3014437/38769437-7219df4e-4003-11e8-8078-351d51994025.png)
> Maybe still worth a try to experiment with the default icon, though.

I confirmed that this is the case. I added a node called About to Rhythm to guarantee the icon show up since About will be at the top.

**Comment by erfajo** [comment]
This is an absolutely ridiculous problem, but nevertheless still present!

@johnpierson Thanks for the idea to have an “About” node, pointing to the package icon to fix a problem that never should occur!

**Comment by vinnividivicci** [comment]
Thanks for this trick guys, I'm using this work-around of having the "About" method and it's displaying the logo correctly:

![image](https://user-images.githubusercontent.com/591242/62806445-dc0f7b00-bac0-11e9-8bb0-1abfa9f8b571.png)

However, if I make the About method invisible in Dynamo the logo reverts back to its old ways... 

Any other way of hiding the "useless" About node in Dynamo while still making the icon work?

**Comment by johnpierson** [comment]
Nope. Not at this time. _At least that I have found in my testing._
